q2_chemistree/plugin_setup.py

```diff
@@ -29,17 +29,16 @@
                                         artifact_format=MGFDirFmt)
 
 PARAMS = {
-    'database': Str % Choices(['all', 'PubChem', 'bio']),
+    'database': Str % Choices(['all', 'pubchem']),
```

Comment:
In general it is better if these parameters can be more verbose about what they are. Abbreviations and contractions are often hard to decipher for new users. Unless it is a universal abbreviation, I suggest removing it and spelling out the parameter names. See for example the suggestions in the scikit-bio coding guidelines:

http://scikit-bio.org/docs/latest/development/coding_guidelines.html#acceptable-abbreviations